src/Resources/app/storefront/src/checkout/swag-paypal.acdc-fields.js

```diff
@@ -205,25 +206,26 @@ export default class SwagPaypalAcdcFields extends SwagPaypalAbstractStandalone {
             const firstInvalidFieldKey = Object.keys(state.fields).find((key) => !state.fields[key].isValid);
             this.fields[firstInvalidFieldKey]?.focus();
 
+            const field = DomAccess.querySelector(this.cardFieldForm, this.options[firstInvalidFieldKey + 'Selector']);
+
+            /**
+             * @deprecated tag:v10.0.0 - will be removed and replaced by `field.scrollIntoView(..)`
+             */
             window.scrollTo({
-                top: this.getScrollOffset(DomAccess.querySelector(this.cardFieldForm, this.options[firstInvalidFieldKey + 'Selector'])),
+                top: this.getScrollOffset(field),
                 behavior: 'smooth',
             });
-        });
-    }
-
-    onApprove(data) {
-        PageLoadingIndicatorUtil.create();
 
-        const input = document.createElement('input');
-        input.setAttribute('type', 'hidden');
-        input.setAttribute('name', 'paypalOrderId');
-        input.setAttribute('value', data.orderID ?? data.orderId);
-
-        this.confirmOrderForm.appendChild(input);
-        this.confirmOrderForm.submit();
+            // field.scrollIntoView({
+            //     behavior: 'smooth',
+            //     block: 'center',
+            // });
```

Comment:
I can't approve with this here.

Reply:
already approved by Max this way in `trunk`, see [comment here](https://github.com/shopware/SwagPayPal/pull/72#discussion_r1933665963)

Reply:
wait. This is 6.7. Why are you not immediately resolving that?